src/components/Landing/Landing.js

```diff
@@ -0,0 +1,104 @@
+            velit porta sit amet. Morbi ut commodo ipsum, quis dignissim dolor.
+            Duis ultricies tincidunt finibus. Nunc a commodo neque. Etiam nec
+            neque vulputate, efficitur massa sit amet, ullamcorper felis. Duis
+            ipsum quam, pretium ut diam id, efficitur eleifend sapien. Phasellus
+            libero dolor, finibus a scelerisque vel, pulvinar ut sem. Proin sit
+            amet purus euismod dui lobortis lobortis vitae sit amet orci.
+            Phasellus iaculis malesuada erat id interdum. Nam non lacinia risus.
+            Fusce sed turpis ac quam condimentum pulvinar. Praesent convallis
+            aliquet leo, quis vestibulum nunc vestibulum id. Nulla facilisi. In
+            id dolor id quam malesuada gravida sed eget nulla. Mauris id tortor
+            at mi varius lobortis nec id augue. Curabitur efficitur augue quis
+            diam convallis, vel dictum tortor pulvinar. Nam facilisis sit amet
+            nisi sit amet consequat. Ut placerat malesuada tristique. Nullam
+            pharetra interdum est venenatis varius. Cras ac pellentesque ex.
+          </p>
+        </div>
+        <div label='/it.png' title='Infrastructure as Code'>
+          <p>
+            Donec condimentum enim at tortor maximus pellentesque. Sed placerat id
+            erat nec bibendum. Etiam ac commodo ipsum. Aenean vitae massa semper,
+            semper elit sit amet, hendrerit nisi. Etiam justo urna, ultrices vitae
+            lobortis sed, semper vel dui. Maecenas sem mauris, viverra vel
+            vestibulum nec, mollis a ex. Etiam quis sodales justo. Sed id nulla
+            dignissim turpis hendrerit lacinia ut sed orci. Nullam sed urna id
+            odio mollis venenatis ut nec odio. Nulla vitae orci lacinia, lobortis
+            nisl sit amet, eleifend orci. Quisque in tempor enim, consectetur
+            auctor orci. Donec condimentum ligula tincidunt velit ultricies
+            pulvinar. Vivamus fringilla, lorem a pellentesque commodo, massa dolor
+            congue sem, a interdum lacus lectus eget nibh. Sed ut dui nisl. Fusce
+            hendrerit leo a leo iaculis imperdiet. Phasellus libero dolor, finibus
+            a scelerisque vel, pulvinar ut sem. Proin sit amet purus euismod dui
+            lobortis lobortis vitae sit amet orci. Phasellus iaculis malesuada
+            erat id interdum. Nam non lacinia risus. Fusce sed turpis ac quam
+            condimentum pulvinar. Praesent convallis aliquet leo, quis vestibulum
+            nunc vestibulum id. Nulla facilisi. In id dolor id quam malesuada
+            gravida sed eget nulla. Mauris id tortor at mi varius lobortis nec id
+            augue. Curabitur efficitur augue quis diam convallis, vel dictum
+            tortor pulvinar.
+          </p>
+        </div>
+      </Tabs>
+    </div>
+    <Clients clients={clients} />
+    <Skills skills={skills} />
+    <h1>Nos derniers posts</h1>
```

Comment:
This should be in the `LastPosts` component if it's not used somewhere else. Even then, seems more relevant.